api/models/Community.js

```diff
@@ -159,7 +159,7 @@ module.exports = bookshelf.Model.extend(merge({
     return Post.query(q => {
       q.select(bookshelf.knex.raw('count(*)'))
       q.join('communities_posts', 'posts.id', 'communities_posts.post_id')
-      q.where({'communities_posts.community_id': this.id, 'active': true})
+      q.where({'communities_posts.community_id': this.id, 'posts.active': true})
```

Comment:
`posts.active` vs `active` was changed here for nested comments work, is this also changed in Group.js?

Reply:
yes